### PR TITLE
COMMERCE-4689 Replace edit link with an icon button

### PR DESCRIPTION
**Pull request by carloslancha**
https://issues.liferay.com/browse/COMMERCE-4689

image

Once we decide what to do with https://issues.liferay.com/browse/COMMERCE-4688 this will look perfect ;)

### CODE REVIEW

File: modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/header/page.jsp [hunk 1]
```diff
@@ -103,11 +103,11 @@ String myWorkflowTasksPortletNamespace = PortalUtil.getPortletNamespace(PortletK
...
-												href="#"
+											<clay:button
+												displayType="unstyled"
+												icon="pencil"
 												id="erc-edit-modal-opener"
-												label='<%= LanguageUtil.get(request, "edit") %>'
+												small="<%= true %>"
 											/>
 
 											<aui:script require="commerce-frontend-js/utilities/eventsDefinitions as events">
```
Comment (carloslancha) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/header/page.jsp:113]: Once https://github.com/liferay-frontend/liferay-portal/pull/354 is merged we'll have support for `propsTransformers` in `clay:button` so we could get rid of this `aui:script` and replace it with a `propsTransformer`.

Could you please guys create a task to track this so we don't forget? (I don't know your workflow). Thanks!

cc @marco-leo

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**